src/app/admin/_components/Event/TagSelect.js

```diff
@@ -1,25 +1,47 @@
-import React from 'react';
+import React, { useState } from 'react';
 
 import CreatableSelect from 'react-select/creatable';
 import { Loading, useGetList, useCreate } from 'react-admin';
 import PropTypes from 'prop-types';
 import { RESOURCES } from '@admin/_lib/consts';
 
 export function TagSelect({ setSelectedTags, defaultValue }) {
+  let newTagsLength;
   const [create] = useCreate();
+  const [value, setValue] = useState(defaultValue);
+  const [error, setError] = useState(false);
   const { data: tags, isLoading } = useGetList(RESOURCES.eventTag);
+
   if (isLoading) return <Loading />;
+
+  const onChange = newTags => {
+    newTagsLength = newTags.reduce((currLen, tag) => currLen + tag.label.length, 0);
+    if (newTagsLength > 16) {
```

Comment:
You do mention about 24 symbols in the error message but here is 16. Looks like a mistake for me

Reply:
Actually it's 16, mistake in error message, I`ll change it